core/src/main/java/de/jplag/logging/ProgressBarType.java

```diff
@@ -6,6 +6,7 @@
 public enum ProgressBarType {
     LOADING("Loading Submissions  ", false),
     PARSING("Parsing Submissions  ", false),
+    HASH_CREATION("Preparing Submissions", false),
```

Comment:
Enum naming not consistent with CLI message text. Do we really create hashes here?